mavisp/mavisp.py

```diff
@@ -235,6 +236,33 @@ def main():
     dataset_tables_path = out_path / 'dataset_tables'
     dataset_tables_path.mkdir(exist_ok=True)
 
+   # Generate a csv file that contains the number of mutations and the date of the run
+    time = strftime("%Y-%m-%d", gmtime())
+    nb_mutations = len(mfs.dataset_table['mutations'].explode())
```

Comment:
seems to me that we're counting the mutations of a protein twice if it is both in simple and ensemble mode, while we should count them once in that case